JavaScript/Indentation Rules - Mappings.tmPreferences

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+	<key>scope</key>
+	<string>
+		source.js meta.mapping,
+		source.js meta.sequence,
+		source.jsx meta.mapping,
+		source.jsx meta.sequence,
+		source.ts meta.mapping,
+		source.ts meta.sequence,
+		source.tsx meta.mapping,
+		source.tsx meta.sequence
```

Comment:
You might be able to simplify this via `(source.js, source.jsx, source.ts, source.tsx) & (meta.mapping, meta.sequence)`, but it won't be equivalent since it does not enforce the same order …

Reply:
Order is what I want to enforce as we need to deal with all the sorts of syntaxes interpolating each other, properly.